plugin/src/main/scala/sbtcompendium/CompendiumPlugin.scala

```diff
@@ -86,13 +72,37 @@ object CompendiumPlugin extends AutoPlugin {
       val (failures, generated) = generateProtocols.separate
 
       failures.foreach {
-        case (id, ProtocolNotFound(_)) => log.error(s"Protocol [${id}] not found in Compendium server")
-        case (id, ex)                  => log.error(s"Unable to generate client for [${id}], unknown error [${ex.getMessage}]")
+        case (id, ProtocolNotFound(_)) => log.error(s"Protocol [$id] not found in Compendium server")
+        case (id, ex)                  => log.error(s"Unable to generate client for [$id], unknown error [${ex.getMessage}]")
       }
 
       generated.flatten
     }
   )
 
+  override def trigger: PluginTrigger = allRequirements
+
   override def projectSettings: Seq[Setting[_]] = defaultSettings
+
+  object autoImport {
```

Comment:
We usually put this in the top, before use it. It'll also reduce some changes

Reply:
I don't know why is this moved to the bottom. Maybe Intellij's auto-formatting made that change.

Reply:
From my experience, Intellij's auto-formatting is not a good friend. scalafmt is enough and at compile time (I'd disable any auto-formatting thing in the IDE)